### PR TITLE
ignored `x-internal` endpoint for neutron rest doc

### PR DESCRIPTION
**Pull request by ye11ow**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
     "clear": "docusaurus clear",
     "serve": "docusaurus serve",
     "write-translations": "docusaurus write-translations",
+    "rest-doc": "redocly bundle external@latest --output external.yaml && redocly build-docs ./external.yaml -o static/rest.html && rm ./external.yaml",
```
Comment (ye11ow) [inline package.json:15]: @jovezhong feel free to add params `redocly build-docs ./external.yaml -o static/rest.html` part . Just make sure you are building with `./external.yaml`

Comment (jovezhong) [reply package.json:15]: better to keep the theme (font,color) under static folder

Comment (jovezhong) [reply package.json:15]: --options=rest_theme.json

**Review by jovezhong — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (ye11ow) [reply package.json:15]: @jovezhong hmm, I'm getting `Unknown argument: options` when running `redocly build-docs ./external.yaml --options=static/rest_theme.json -o static/rest.html`

I cannot find this options in their doc as well https://redocly.com/docs/cli/commands/build-docs/
I'm using
```
> redocly --version
1.0.2
```

Comment (jovezhong) [reply package.json:15]: I am using `redoc-cli build`

Comment (jovezhong) [reply package.json:15]: https://redocly.com/docs/api-reference-docs/configuration/theming/#example-theme you may have to add it to your redocly.yaml

Comment (ye11ow) [reply package.json:15]: `redoc-cli` is deprecated so I updated the theme https://github.com/timeplus-io/docs/pull/87/commits/f3dad891b377626bd414a81953e2f0280c69bfc9
```
   ┌───────────────────── DEPRECATED ─────────────────────┐
   │                                                      │
   │   This package is deprecated.                        │
   │                                                      │
   │   Use `npx @redocly/cli build-docs <api>` instead.   │
   │                                                      │
   └──────────────────────────────────────────────────────┘
```

Later on we can just `yarn rest-doc`

**Review by jovezhong — APPROVED** [review]

**Comment by jovezhong** [comment]
(Jove Github Bot) moved this ticket out of the GitHub project(up to 1200 tickets for one project).

**Pull request merged**